basis/graph/configured_node.py

```diff
@@ -49,15 +47,15 @@ class InputDefinition(FrozenPydanticBase):
     name: str
 
     description: str = None
-    schema_or_name: Union[str, Schema] = None
+    schema_or_name: str = None
     required: bool
 
 
 class OutputDefinition(FrozenPydanticBase):
     port_type: PortType
     name: str
     description: str = None
-    schema_or_name: Union[str, Schema] = None
+    schema_or_name: str = None
```

Comment:
We could rename this to the more accurate `schema_name` (or just `schema`). Does the protocol reference this field?

Reply:
Yeah let's rename here